java/openmldb-taskmanager/src/test/java/com/_4paradigm/openmldb/taskmanager/TestJobIdGenerator.java

```diff
@@ -0,0 +1,47 @@
+package com._4paradigm.openmldb.taskmanager;
+
+import com._4paradigm.openmldb.taskmanager.dao.JobIdGenerator;
+import org.testng.Assert;
+import org.testng.annotations.Ignore;
+import org.testng.annotations.Test;
+
+import java.util.concurrent.*;
+
+public class TestJobIdGenerator {
+    public CopyOnWriteArrayList<Integer> jobIdSet = new CopyOnWriteArrayList<>();
+    public int num = 0;
```

Comment:
This is not thread-safe. Each thread may uses its internal num instead of updating the same `num` in class.